api/src/util/logging.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import traceback
+
+
+def init_logging():
+    try:
+        logger = logging.getLogger("API")
+        if logger.hasHandlers():
+            logger.handlers.clear()
+        logger.setLevel("INFO")
+        formatter = logging.Formatter("%(levelname)s:%(asctime)s %(message)s")
+        handler = logging.StreamHandler()
+        handler.setFormatter(formatter)
+        handler.setLevel("INFO")
+        logger.addHandler(handler)
+        logger.info("Logger is configured")
+    except (ValueError, TypeError) as e:
```

Comment:
Don't think we should handle exceptions here. Rather let them boil up and crash the entire app :)